modules/ROOT/pages/dw-multipart.adoc

```diff
@@ -15,7 +15,7 @@ DataWeave script.
 | Name  | Description
 | xref:dw-multipart-functions-field.adoc[field] | Creates a `MultipartPart` data structure using the specified part name,
 input content for the part, format (or mime type), and optionally, file name.
-| xref:dw-multipart-functions-file.adoc[file] | Creates a `MultipartPart` data structure from a resource file.
```

Comment:
MDA: This is in the source but doesn't appear in the output. TBD if it getting generated.